tests/check_codeblocks.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Script that checks if the Code::Blocks project is complete.
+# TODO: If the project is incomplete generate a patch file to make it complete.
```

Comment:
Generating the patch is nice, but I feel that this PR is already very useful if we just get the warning; we always added those files manually anyway.

I would be okay with just not having this TODO item here at all.

Reply:
I think the TODO is still useful if someone with more Bash knowledge stumbles upon it.